test/cli_run_volume_test.go

```diff
@@ -170,6 +172,53 @@ func (suite *PouchRunVolumeSuite) TestRunWithVolumesFromWithDupclicate(c *check.
 	c.Assert(volumeFound, check.Equals, true)
 }
 
+func (suite *PouchRunVolumeSuite) TestRunWithVolumesFromDifferentSources(c *check.C) {
+	imageWithVolume := "registry.hub.docker.com/shaloulcy/busybox:with-volume"
```

Comment:
Please add a `todo` here, we'd better build image explicitly when `build` CMD is supported in pouch.

Reply:
fine